scripts/send-bundles-sizes-differences.js

```diff
@@ -0,0 +1,181 @@
+const { execSync } = require('child_process')
+const { runMain, timeout } = require('./lib/execution-utils')
+const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('./lib/secrets')
+
+const header = 'Bundles Sizes Evolution'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏more explicit naming
```suggestion
const prCommentHeader = 'Bundles Sizes Evolution'
```